chrome/tabbar/tabbar.css

```diff
@@ -480,7 +480,8 @@
 }
 
 #tabbrowser-tabs[overflow] .tabbrowser-tab[first-visible-tab]:not([pinned]),
-#tabbrowser-tabs[overflow] .tabbrowser-tab[pinned] + .tabbrowser-tab:not([pinned])
+#tabbrowser-tabs[overflow] .tabbrowser-tab[pinned] + .tabbrowser-tab:not([pinned]),
+#tabbrowser-tabs:not([overflow]) .tabbrowser-tab:first-child
```

Comment:
Should be `#tabbrowser-tabs:not([overflow]) .tabbrowser-tab[first-visible-tab]`.

Reply:
Done. Is there any difference between the two or is this just for code style?